adapter/mysql/adapter.go

```diff
@@ -2,12 +2,13 @@ package mysql
 
 import (
 	"fmt"
-	"github.com/go-sql-driver/mysql"
-	"github.com/jmoiron/sqlx"
-	"github.com/sue445/plant_erd/db"
 	"sort"
 	"strconv"
 	"strings"
+
+	"github.com/go-sql-driver/mysql"
```

Comment:
Why is the order here changed?

Reply:
It seemed to be formatted by my editor's extension (perhaps `vscode-go`) automatically.
Should I revert it?

Reply:
I see. This is OK as it is